lib/tasks/number_of_people.rake

```diff
@@ -1,69 +1,66 @@
-# frozen_string_literal: true
-
 namespace :db do
+
+  # Execute given block on non-valid rows
+  def on_differing_rows(model_class)
```

Comment:
La logique de vérification de l'erreur est fusionnée dans une méthode.
Par contre je ne suis pas certain que le design pattern utilisé (block method ?), ainsi que son emplacement dans l'arborescence des namespaces soit le meilleur choix.

Reply:
Oui, effectivement, définir une méthode dans une tâche rake est plus risqué que ce qu'on pourrait penser. Cet article l'explique bien : https://kevinjalbert.com/defined_methods-in-rake-tasks-you-re-gonna-have-a-bad-time/

Tu pourrais extraire les méthodes (celle-ci, `check_number_of_people` et `fix_number_of_people`) dans un service ou un module ?

Pour le block, c'est également la seule solution que je vois. Juste pour ta curiosité, une variante serait possible où la méthode retournerait un `Enumerator::Lazy` au lieu de prendre un block, mais c'est quasiment identique.